src/components/Loading/Loading.js

```diff
@@ -4,9 +4,9 @@ import PropTypes from 'prop-types';
 import { prefixedId } from '../../utils';
 
 export const Loading = ({ text, id }) => (
-  <div id={id} className="wizard-pf-process blank-slate-pf" key={prefixedId(id, 'progress') || 'progress'}>
```

Comment:
can you review usage of this please?

Reply:
oh, sure, i will

Reply:
So looking at PF loading wizard, it is using `blank-slate-pf` without `wizard-pf-process` https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/src/components/Wizard/Stories/mockWizardItems.js#L4 so I kept only  `blank-slate-pf`